build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -1997,6 +1997,10 @@ message CacheCapabilities {
   // [BatchUpdateBlobs][build.bazel.remote.execution.v2.ContentAddressableStorage.BatchUpdateBlobs]
   // requests.
   repeated Compressor.Value supported_batch_update_compressors = 7;
+
+  // Maximum size a single CAS entry can have in the remote cache system.
+  // A value of 0 means no limit is set
+  int64 max_cas_entry_size_bytes = 8;
```

Comment:
I'd like to see something here describing how implementations should interpret this limit:

1. SHOULD implementations not attempt to upload a blob larger than this? MUST they not? (note the latter is a breaking change and would require a version bump)
2. MAY implementations return a blob larger than this size?

Reply:
Good points

I'm for hint semantics for the limit here, i.e.
1. Implementations SHOULD not attempt to upload blobs over the threshold. If they do, it is implementation defined what happens. E.g. a remote cache server may accept the request anyway, or reject it as invalid.
2. Implementations MAY return blobs larger than the threshold. The threshold may be set after the CAS in use was created and have larger blobs, in which case we don't want to enforce skipping those blobs.
3. The value 0 means the server implementations do not have special requirements when it comes to the size of CAS entries.

Reply:
@tjgq do you agree with the new description?
@bergsieker any issues from your side?

Reply:
LGTM from the Bazel side, but I'd suggest letting it linger for a week or so, to give other implementors a chance to comment.